recipes/pgcgap/meta.yaml

```diff
@@ -16,50 +16,54 @@ build:
 
 requirements:
   host:
-    - perl
+    - perl 5.26.2
```

Comment:
This will make it incompatible with everything else in bioconda. Are you sure you want to do this?

Reply:
Thanks for your suggestion, I had removed the version of Perl and started a new test to see if it could pass the test.  One more thing, I wonder when can we use the mambabuild.  I am very expecting the new builder because it is painful to use the old one for recipes with a lot of dependency packages.

Reply:
Thanks for your suggestion, I had removed the version of Perl and started a new test to see if it could pass the test.  One more thing, I wonder when can we use the mambabuild.  I am very expecting the new builder because it is painful to use the old one for recipes with a lot of dependency packages.

Reply:
I've just switched over to mambabuild and will use this as the test PR to ensure that's fully working :)

Reply:
Thanks for your kind help! I am honored that this PR can be used for mambabuild testing and hope it will be successful.